docs/design/StratisSoftwareDesign.lyx

```diff
@@ -6933,5 +6933,445 @@ Based on looking at the existing available building blocks, the best option
  provides.
 \end_layout
 
+\begin_layout Section
+Encryption implementation details
+\end_layout
+
+\begin_layout Standard
+Stratis uses libcryptsetup to manage FDE (full disk encryption) for each
+ block device added to an encrypted pool.
+\end_layout
+
+\begin_layout Subsection
+On-disk format
+\end_layout
+
+\begin_layout Standard
+Stratis utilises the LUKS2 encryption format as the basis for the encryption
```

Comment:
^utilises^uses --- I don't always agree with "Strunk and White", but on this word I think they're right.

Reply:
Resolved.